2022-03-04 12:10:31,002 INFO    MainThread:670390 [wandb_setup.py:_flush():71] setting env: {}
2022-03-04 12:10:31,002 INFO    MainThread:670390 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-04 12:10:31,002 INFO    MainThread:670390 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-04 12:10:31,002 INFO    MainThread:670390 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220304_121031-mrr6o7vh/logs/debug.log
2022-03-04 12:10:31,002 INFO    MainThread:670390 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220304_121031-mrr6o7vh/logs/debug-internal.log
2022-03-04 12:10:31,002 INFO    MainThread:670390 [wandb_init.py:init():404] calling init triggers
2022-03-04 12:10:31,002 INFO    MainThread:670390 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-03-04 12:10:31,002 INFO    MainThread:670390 [wandb_init.py:init():460] starting backend
2022-03-04 12:10:31,002 INFO    MainThread:670390 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-04 12:10:31,010 INFO    MainThread:670390 [backend.py:ensure_launched():216] starting backend process...
2022-03-04 12:10:31,013 INFO    MainThread:670390 [backend.py:ensure_launched():222] started backend process with pid: 670399
2022-03-04 12:10:31,014 INFO    MainThread:670390 [wandb_init.py:init():469] backend started and connected
2022-03-04 12:10:31,016 INFO    MainThread:670390 [wandb_init.py:init():533] updated telemetry
2022-03-04 12:10:31,028 INFO    MainThread:670390 [wandb_init.py:init():563] communicating current version
2022-03-04 12:10:31,475 INFO    MainThread:670390 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-04 12:10:31,476 INFO    MainThread:670390 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-04 12:10:32,073 INFO    MainThread:670390 [wandb_init.py:init():606] starting run threads in backend
2022-03-04 12:10:34,739 INFO    MainThread:670390 [wandb_run.py:_console_start():1810] atexit reg
2022-03-04 12:10:34,739 INFO    MainThread:670390 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-04 12:10:34,740 INFO    MainThread:670390 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-04 12:10:34,741 INFO    MainThread:670390 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-04 12:10:34,741 INFO    MainThread:670390 [wandb_init.py:init():633] run started, returning control to user process
2022-03-04 12:10:47,615 INFO    MainThread:670390 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-04 12:10:47,616 INFO    MainThread:670390 [wandb_run.py:_restore():1752] restore
2022-03-04 12:10:50,266 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-04 12:10:51,683 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-04 12:10:52,083 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15757
}

2022-03-04 12:10:52,184 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15757
}

2022-03-04 12:10:52,287 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15757
}

2022-03-04 12:10:52,388 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15757
}

2022-03-04 12:10:52,491 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15757
  total_bytes: 15757
}

2022-03-04 12:10:52,593 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15757
  total_bytes: 15757
}

2022-03-04 12:10:52,696 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15757
  total_bytes: 15757
}

2022-03-04 12:10:52,799 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15757
  total_bytes: 15757
}

2022-03-04 12:10:52,901 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15757
  total_bytes: 15757
}

2022-03-04 12:10:53,142 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15757
  total_bytes: 15757
}

2022-03-04 12:10:53,747 INFO    MainThread:670390 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15757
  total_bytes: 15757
}
local_info {
}

2022-03-04 12:10:54,813 INFO    MainThread:670390 [wandb_run.py:_append_history():2130] rendering history
2022-03-04 12:10:54,813 INFO    MainThread:670390 [wandb_run.py:_append_summary():2085] rendering summary
2022-03-04 12:10:54,814 INFO    MainThread:670390 [wandb_run.py:_append_files():2180] logging synced files
